src/Xamarin.Android.Build.Tasks/Tasks/GenerateJavaStubs.cs

```diff
@@ -226,8 +226,7 @@ void Run (DirectoryAssemblyResolver res)
 			foreach (var kvp in managedConflicts) {
 				Log.LogCodedWarning (
 					"XA4214",
-					"The managed type `{0}` exists in multiple assemblies: {1}. " +
-					"Please refactor the managed type names in these assemblies so that they are not identical.",
+					Properties.Resources.XA4214A,
 					kvp.Key,
 					string.Join (", ", kvp.Value));
 				Log.LogCodedWarning ("XA4214", "References to the type `{0}` will refer to `{0}, {1}`.", kvp.Key, kvp.Value [0]);
```

Comment:
This line should also be translated.  *All* instances of XA4214 should be handled in the same PR, in part to "demonstrate" how "identical codes but different messages" should be handled.